src/index.html

```diff
@@ -11,7 +11,211 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="#" method="post">
+      <fieldset class="group-fieldset group-pesonal">
+        <legend>Personal information</legend>
+        <label
+          class="label"
+          for="surname"
+        >
+          Surname:
+          <input
+            class="input"
+            type="text"
+            id="surname"
+            name="surname"
+            autocomplete="off"
+          >
+        </label>
+        <label
+          class="label"
+          for="name"
+        >
+          Name:
+          <input
+            class="input"
+            type="text"
+            id="name"
+            name="name"
+            autocomplete="off"
+          >
+        </label>
+        <label
+          class="label"
+          for="age"
+        >
+          How old are You?
+          <input
+            class="input"
+            type="number"
+            id="age"
+            value="12"
+            name="age"
+            min="1"
+            max="100"
+          >
+        </label>
+        <label
+          class="label"
+          for="birth"
+        >
+          Full date of birth:
+          <input
+            class="input"
+            type="date"
+            id="birth"
+            placeholder="mm/dd/yyyy"
+            name="birthDate"
+          >
+        </label>
+        <label
+          class="label"
+          for="accept"
+        >
```

Comment:
I would say that labels with 2 attributes are quite short and clear, so I would have kept them on a single line
```suggestion
        <label class="label" for="accept">
```